blueprints/ember-cli-addon-docs/index.js

```diff
@@ -63,9 +63,39 @@ module.exports = {
     }
 
     if (!hasPlugins) {
+      return this._chooseAddonsToInstall()
```

Comment:
Looks like there's a lint failure because the `return` below this one is dead code now